airbyte-integrations/connectors/source-stripe/source_stripe/source.py

```diff
@@ -101,7 +101,8 @@ def get_updated_state(self, current_stream_state: MutableMapping[str, Any], late
     def request_params(self, stream_state=None, **kwargs):
         stream_state = stream_state or {}
         params = super().request_params(stream_state=stream_state, **kwargs)
-        params["created[gte]"] = stream_state.get(self.cursor_field)
+        if stream_state:
```

Comment:
perhaps you should also check if `cursor_field` exist?
```suggestion
        if stream_state and stream_state.get(self.cursor_field):
```

Reply:
Done.